src/libstore/build.cc

```diff
@@ -59,10 +59,12 @@
 /* chroot-like behavior from Apple's sandbox */
 #if __APPLE__
     #define SANDBOX_ENABLED 1
-    #define DEFAULT_ALLOWED_IMPURE_PREFIXES "/System/Library /usr/lib /dev /bin/sh"
+    #define DEFAULT_ALLOWED_IMPURE_PREFIXES "/System/Library /usr /dev /bin/sh"
```

Comment:
As discussed on IRC, these shouldn't be necessary any more, right? The paths added by the pre-build-hook are not subject to the allowed prefixes check